### ISSUE TITLE
Sudden FPS Drops to 0fps that last about 5 seconds

### ISSUE THREAD

**Issue by Kccool2** [The-1-7-10-Pack/The-1.7.10-Pack#245]
Hi, loving the pack, the one and only pack I play, but for some reason, I've started to get sudden FPS drops to 0 fps that last about 5 seconds, whenever I open any machine or chests inventory (right click menu) and then quickly open another chest/machine inventory.

I've managed to recreate this after extensive trial and tribulation and then consequently finding out the issue happens whenver I go between machines/chests/anything with a right click menu quickly. 

I installed optifine after this began happening and the issue persists. I have no extra mods, and this only started happening after I upgraded to 7.2, then I downgraded to 7.1 and the issue again, still persisted. 

I didn't know where else to go to find help about this, as every forum thread I read either points to optifine, or fastcraft (which is included in the pack of course) and neither of those are helping apparently. 

Again, this is the only pack I play, and want to play, so I'm really hoping we can sort this! 

Love your pack, 

Kc

**Comment by The-1-7-10-Pack** [comment]
Since when do you play in the modpack?
Did you try to completely reinstall? (delete the whole modpack's directory after you backup your saves)

**Comment by Kccool2** [comment]
I have been playing this modpack since version 1.7. When I downgraded I did
completely reinstall, in the sense of choosing the reinstall pack option,
but I didn't delete the whole modpack directory no. I will try that and
then reinstall, play and see if the issue persists.

I will get back to you and let you know if the issue still happens.

Thank you very much for your speedy reply, it's very appreciated!
